### PR TITLE
update zh-translation: content/zh/docs/tasks/tools/install-minikube.md

### PR DESCRIPTION
**Pull request by gorda**
Update base on [master#en](https://github.com/kubernetes/website/blob/master/content/en/docs/tasks/tools/install-minikube.md)

### CODE REVIEW

**Comment by gorda** [comment]
[Preview](https://deploy-preview-19626--kubernetes-io-master-staging.netlify.com/zh/docs/tasks/tools/install-minikube/) of changes to this pr

File: content/zh/docs/tasks/tools/install-minikube.md [hunk 1]
```diff
@@ -1,110 +1,482 @@
...
 
 {{% capture overview %}}
 
-本页面讲述如何安装 Minikube。
 <!--
-This page shows how to install Minikube.
+This page shows you how to install [Minikube](/docs/tutorials/hello-minikube), a tool that runs a single-node Kubernetes cluster in a virtual machine on your personal computer.
 -->
 
+本页面讲述如何安装 [Minikube](/docs/tutorials/hello-minikube)，该工具用于在您电脑中的虚拟机上运行一个单机的 Kubernetes 集群。
```
Comment (tengqm) [inline content/zh/docs/tasks/tools/install-minikube.md]:
```suggestion
本页面讲述如何安装 [Minikube](/docs/tutorials/hello-minikube)，该工具用于在您电脑中的虚拟机上运行一个单节点的 Kubernetes 集群。
```

File: content/zh/docs/tasks/tools/install-minikube.md [hunk 2]
```diff
@@ -1,110 +1,482 @@
...
+If you're using the `none` driver in Debian or a derivative, use the `.deb` packages for
+Docker rather than the snap package, which does not work with Minikube.
+You can download `.deb` packages from [Docker](https://www.docker.com/products/docker-desktop).
+-->
+
+Minikube 还支持使用一个 `--vm-driver=none` 选项，让 Kubernetes 组件运行在主机中，而不是在 VM 中。
+使用这种驱动方式需要 [Docker](https://www.docker.com/products/docker-desktop) 和 Linux 环境，但不需要 hypervisor。
+
+如果你在 Debian 系的 OS 中使用了 `none` 这种驱动方式，请使用 `.deb` 包安装 Docker，不要使用 snap 包的方式，Minikube 不支持这种方式。
+你可以从 [Docker](https://www.docker.com/products/docker-desktop) 下载 `.deb` 包
```
Comment (tengqm) [inline content/zh/docs/tasks/tools/install-minikube.md]:
```suggestion
你可以从 [Docker](https://www.docker.com/products/docker-desktop) 下载 `.deb` 包。
```

File: content/zh/docs/tasks/tools/install-minikube.md [hunk 3]
```diff
@@ -1,110 +1,482 @@
...
+
+{{< caution >}}
+<!--
+The `none` VM driver can result in security and data loss issues.
+Before using `--vm-driver=none`, consult [this documentation](https://minikube.sigs.k8s.io/docs/reference/drivers/none/) for more information.
+-->
+
+`none` VM 驱动方式存在导致安全和数据丢失的问题。
```
Comment (tengqm) [inline content/zh/docs/tasks/tools/install-minikube.md]: 漏了一句

**Pull request merged**